### PR TITLE
ACE fix framing behaviors

### PR DESCRIPTION
**Pull request by carolhmj**

### CODE REVIEW

File: dist/preview release/what's new.md [hunk 1]
```diff
@@ -155,6 +155,7 @@
 - Set default tangents for new keys on ACE and fix their behavior ([carolhmj](https://github.com/carolhmj))
 - Let the playhead move past the last animation key on ACE ([carolhmj](https://github.com/carolhmj))
 - Improve zooming on ACE ([carolhmj](https://github.com/carolhmj))
+- Improve framing on ACE ([carolhmj](https://github.com/carolhmj))
```
Comment (deltakosh) [inline dist/preview release/what's new.md:158]: We should not mention ACE updates as ACE is new. The only line about ACE should be the one in the top section ;)

Comment (carolhmj) [reply dist/preview release/what's new.md]: Removed the line!

Comment (deltakosh) [reply dist/preview release/what's new.md]: All of them actually ;)

Comment (RaananW) [reply dist/preview release/what's new.md]: @carolhmj - want to take care of this in this PR, or should I merge this and you'll submit a new one later?

**Pull request merged**